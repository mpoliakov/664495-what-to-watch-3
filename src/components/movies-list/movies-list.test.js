import React from "react";
import renderer from "react-test-renderer";
import MoviesList from "./movies-list.jsx";

it(`MoviesList is rendered correctly`, () => {
  const films = [{
    id: `2fc32fa3-e5d4-4e7d-8f58-397cc8c9bba7`,
    title: `Alien`
  },
  {
    id: `7c8b88b2-091b-4cd1-9589-3c88b77a9104`,
    title: `Home alone`
  }];

  const component = renderer.create(<MoviesList films={films} />).toJSON();

  expect(component).toMatchSnapshot();
});
