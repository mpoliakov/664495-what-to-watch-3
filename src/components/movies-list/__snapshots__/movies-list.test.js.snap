// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoviesList/> is rendered correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video>
        <source />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Alien
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video>
        <source />
      </video>
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Home alone
      </a>
    </h3>
  </article>
</div>
`;
